src/Plugin/dgi_migrate_alter/foxml/DgisNodesAlter.php

```diff
@@ -0,0 +1,287 @@
+
+    $process['field_record_information'][3]['values']['_field_record_creation_date_single'][0]['query'] =
+      'mods:recordInfo/mods:recordCreationDate[not(@point)] or mods:originInfo/mods:dateCreated[not(@point)]';
+    $process['field_record_information'][3]['values']['_field_record_creation_date_start'][0]['query'] =
+      'mods:recordInfo/mods:recordCreationDate[@point="start"] or mods:originInfo/mods:dateCreated[@point="start"]';
+    $process['field_record_information'][3]['values']['_field_record_creation_date_end'][0]['query'] =
+      'mods:recordInfo/mods:recordCreationDate[@point="end"] or mods:originInfo/mods:dateCreated[@point="end"]';
+    $process['field_record_information'][3]['values']['field_record_information_note'][] = [
+      'plugin' => 'single_value',
+    ];
+    $process['field_record_information'][3]['values']['field_record_information_note'][] = [
+      'plugin' => 'callback',
+      'callable' => 'array_filter',
+    ];
+    $process['field_record_information'][3]['values']['field_record_information_note'][] = [
+      'plugin' => 'null_coalesce',
+    ];
+    $process['field_related_item_paragraph'][3]['values']['field_related_item_genre'][] = [
+      'plugin' => 'single_value',
+    ];
+    $process['field_related_item_paragraph'][3]['values']['field_related_item_genre'][] = [
+      'plugin' => 'callback',
+      'callable' => 'array_filter',
+    ];
+    $process['field_related_item_paragraph'][3]['values']['field_related_item_genre'][] = [
+      'plugin' => 'null_coalesce',
+    ];
+
+    $process['_rights_statement'] = $process['_resource_type'];
+    $process['_rights_statement'][0]['query'] = 'mods:accessCondition[@type="rights statement"]';
+    $process['_rights_statement'][4]['values']['_vid'][0]['default_value'] = 'rights_statements';
+
+    $process['_unspecified_rights_statement'] = $process['_unspecified_resource_type'];
+    $process['_unspecified_rights_statement'][1]['use_as_key'] = '@_rights_statement';
+    $process['_unspecified_rights_statement'][2]['values']['_vid'][0]['default_value'] = 'rights_statements';
+
+    $process['field_rights_statement'] = $process['field_resource_type'];
+    $process['field_rights_statement'][0]['source'] = [
+      '@_rights_statement',
+      '@_unspecified_rights_statement',
+    ];
+
+    $process['field_publication_genre'][3] = [
+      'plugin' => 'single_value',
+    ];
+
+    array_splice($process['field_publication_genre'], 4, 0, [
+      [
+        'plugin' => 'callback',
+        'callable' => 'array_filter',
+      ],
+      [
+        'plugin' => 'null_coalesce',
+      ],
+    ]);
+
+    $process['field_publication_title'][3] = [
+      'plugin' => 'single_value',
+    ];
+
+    array_splice($process['field_publication_title'], 4, 0, [
+      [
+        'plugin' => 'callback',
+        'callable' => 'array_filter',
+      ],
+      [
+        'plugin' => 'null_coalesce',
+      ],
+    ]);
+
```

Comment:
Looks like `field_publication_number` may need to be updated, from the dump:

```
 'field_publication_number' => array (8) [
        0 => array (5) [
            'plugin' => string (37) "dgi_migrate.process.xml.context_query"
            'missing_behavior' => string (12) "skip_process"
            'source' => string (11) "@_mods_node"
            'xpath' => string (12) "@_mods_xpath"
            'query' => string (61) "mods:relatedItem[@type="host"]/mods:titleInfo/mods:partNumber"
        ]
...
```
From the MMS: `mods:relatedItem[@type="host"]/mods:part/mods:detail[@type="issue"]/mods:number`.

Reply:
Fixed.